client/src/components/TextInputComponent/TextInput.css

```diff
@@ -0,0 +1,18 @@
+@font-face {
+  font-family: Raleway;
+  src: url("https://fonts.googleapis.com/css?family=Raleway&display=swap");
+}
```

Comment:
This should be set in the `App.css` file now, once you update your branch with `develop`

Reply:
@Creizai looks like maybe you missed this one?